### ISSUE TITLE
Links with trailing slash don't work

### ISSUE THREAD

**Issue by liquidvisual** [nuxt-community/nuxtent-module#120]
First of all, great library! Thanks for making this.

I've noticed when navigating to pages generated with markdown, they become invalid with a trailing slash. Pages don't exhibit this behaviour though.

eg:

```
localhost:3000/hello-world     --> Works (content/hello-world.md)
localhost:3000/hello-world/    --> Fails (content/hello-world.md)
```

although:

```
localhost:3000/about/  --> Works (pages/about.vue)
```

<div align="right"><sub><em>This feature request is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt-community/nuxtent-module/issues/c96">#c96</a>)</em></sub></div>

**Comment by alidcastano** [comment]
Ye the content path must not be getting matched if it includes a trailing slash

**Comment by liquidvisual** [comment]
Is there any way to fix this?

**Comment by dzabrzenski** [comment]
You can remove trailing slash from router path in your asyncData function.

For example

https://github.com/nuxt-community/nuxtent-module/blob/master/examples/multiple-content-types/pages/projects/_slug/index.vue#L12


Here you

**Comment by dzabrzenski** [comment]
You can remove trailing slash from path passing to your asyncData function.

For example

https://github.com/nuxt-community/nuxtent-module/blob/master/examples/multiple-content-types/pages/projects/_slug/index.vue#L12

replace this line with:

`await app.$content('/projects').get(route.path.replace(/\/$/, '').toLowerCase())`

**Comment by cesasol** [comment]
FIxed in c86e33aeb6d9d43c70bdd18ded11ff566dd37794
